### PR TITLE
Updated to delta

### PR DESCRIPTION
**Pull request by TowleT**
Updated to delta

### CODE REVIEW

File: api/html5_touch.js [hunk 1]
```diff
@@ -122,9 +122,9 @@ Touch.prototype.radiusY = { };
 
 /**
 *
-* the angle (in degrees) that the ellipse described by radiusX and radiusY is rotated 
+* The angle (in degrees) that the ellipse described by radiusX and radiusY is rotated 
 * clockwise about its center; 0 if no value is known. The value must be greater than 
-* or equal to 0 and less than 90. <br /><br />
+* or equal to 0 and less than 90. Corrected grammar and typos.<br />
```
Comment (jeffheifetz) [inline api/html5_touch.js:126]: Please remove this addition

File: api/html5_deviceMotion.js [hunk 2]
```diff
@@ -219,7 +219,7 @@ DeviceMotionEvent = {
 	
 	/**
 	 * @type Acceleration
-	 * @desc Implementations that are unable to provide acceleration data without the effect of gravity (due, for example, to the lack of a gyroscope) may instead supply the acceleration including the effect of gravity. This is less useful in many applications but is provided as a means of providing best-effort support. 
+	 * @desc Implementations that are unable to provide acceleration data without the effect of gravity (due, for example, to the lack of a gyroscope) may instead supply the acceleration including the effect of gravity. This is less useful in many applications but is provided as a means of providing best-effort support. Not what this means to a developer??
```
Comment (jeffheifetz) [inline api/html5_deviceMotion.js]: Please remember to remove this comment.

Comment (TowleT) [inline api/html5_deviceMotion.js]: I will remove this comment.

**Comment by jeffheifetz** [comment]
I believe this change should be from your delta to blackberry delta. Please close this request and open one like that.

**Comment by TowleT** [comment]
Refactored in pull request #142

**Pull request closed without merging**